helm/values.yaml

```diff
@@ -72,6 +72,10 @@ logConfig:
       enabled: false
 
 config-bootstrapper:
-  job:
-    prefix: attribute
-  configurationCommands: {}
\ No newline at end of file
+  jobs:
```

Comment:
Do you need to update the config bootstrapper dependency version somewhere?

Reply:
Yes, this pr and https://github.com/hypertrace/entity-service/pull/77 needs to be updated with latest config-bootstrapper chart version once 
https://github.com/hypertrace/config-bootstrapper/pull/33 is merged

Reply:
ok.